protocol/messenger_backup.go

```diff
@@ -308,6 +327,46 @@ func (m *Messenger) backupCommunities(ctx context.Context, clock uint64) ([]*pro
 	return backupMessages, nil
 }
 
+func (m *Messenger) backupChats(ctx context.Context, clock uint64) []*protobuf.Backup {
+	var oneToOneAndGroupChats []*protobuf.SyncChat
+	m.allChats.Range(func(chatID string, chat *Chat) (shouldContinue bool) {
+		if !chat.Active || (!chat.OneToOne() && !chat.PrivateGroupChat()) {
```

Comment:
@jrainville don't we need to backup inactive chats as well? I guess that `Active` flag is `false` if the chat is removed (soft deleted), if so we should backup such chats as well otherwise those removed chats may appear again after recovering.

Reply:
Updated. I added the Active property. I also added a clock, so that we only sync the latest